Backend/tests/mock_data.json

```diff
@@ -1,14 +1,26 @@
 [
   {
-   "email" : "test@mentii.me",
-   "password" : "iameight",
-   "activationId" : "12345",
-   "active": "T"
+    "email" : "test@mentii.me",
+    "password" : "iameight",
+    "activationId" : "12345",
+    "active": "T"
   },
   {
-   "email" : "test2@mentii.me",
-   "password" : "iameight2",
-   "activationId" : "abcde",
-   "active": "F"
+    "email" : "test2@mentii.me",
+    "password" : "iameight2",
+    "activationId" : "abcde",
+    "active": "F"
+  },
+  {
```

Comment:
I think some of the tests use the passwords so if ya change them you'll have to update them accordingly